packages/databyss-ui/components/SearchContent/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import {
+  Text,
+  View,
+  BaseControl,
+  Grid,
+  Icon,
+} from '@databyss-org/ui/primitives'
+import { widthVariants } from '@databyss-org/ui/theming/views'
+
+export const SearchResultsContainer = ({ children }) => (
+  <View mb="medium" maxWidth={widthVariants.content.maxWidth}>
```

Comment:
why not `widthVariant="content"`?

Reply:
Same reason as with the other component, it doesn't get translated to pixels anymore